### PR TITLE
directory: skip owners/peers if they do not have bmo_id (bug 1936373)

### PR DESCRIPTION
**Pull request by zzzeid**
- add aliases to main config fixture
- update tests

### CODE REVIEW

File: tests/test_config.py [hunk 1]
```diff
@@ -299,9 +299,7 @@ def test_clean_added_user_no_bmo_id_with_refresh(
 
     assert file_config.config["people"] == config["people"]
     file_config.config["modules"][0]["owners"].append({"nick": "TLMC"})
-    file_config.config["modules"][0]["submodules"][0]["owners"].append(
-        {"nick": "TLMC"}
-    )
+    file_config.config["modules"][0]["submodules"][0]["owners"].append({"nick": "TLMC"})
```
Comment (zzzeid) [inline tests/test_config.py:302]: Hmm, interesting that this change popped up here. There might be a divergence between the tests and `make format`

Comment (zzzeid) [reply tests/test_config.py:302]: I'll file a separate bug for this, the tests directory is missing from the test lint paths, but is present in the format command.

File: src/mots/directory.py [hunk 2]
```diff
@@ -129,7 +129,11 @@ def peers_and_owners(self):
         peers_and_owners = set()
         for module in modules_and_submodules:
             peers_and_owners.update(
-                [person["bmo_id"] for person in module.peers + module.owners]
+                [
+                    person["bmo_id"]
+                    for person in module.peers + module.owners
+                    if "bmo_id" in person
+                ]
```
Comment (cgsheeh) [inline src/mots/directory.py:136]: This essentially means that a peer/owner without a BMO ID wouldn't be returned by `Directory.peers_and_owners`, correct? Maybe we could document that in the docstring, or somehow indicate that this is returning only peers/owners that are on BMO. Is there a way to get all peers/owners that doesn't require a `bmo_id`?

**Review by cgsheeh — DISMISSED** [review]

File: src/mots/directory.py (hunk 2 above)
Comment (zzzeid) [reply src/mots/directory.py:136]: Yes, this is the same behaviour in the CLI today, and was discussed in the original ask. I'll add a comment. Currently there is only one "alias" (and adding any others would require modifying enforcement mechanisms). The main way this method is used is to link things back to an actual person (i.e., bugzilla ID), so I don't foresee, at least in the current way that aliases are implemented, the need for this to include aliases as well.

**Review by cgsheeh — APPROVED** [review]

**Pull request merged**